### ISSUE TITLE
Copy-and-Paste Characters can crash Javascript content loads

### ISSUE THREAD

**Issue by ewr** [SCPR/SCPRv4#237]
A site producer was having an issue where the homepage content aggregator wouldn't load. Browser was complaining of an invalid EOF.

Turns out it was an inadvertent invisible unicode line break character:

![screen shot 2015-02-13 at 10 19 40 am](https://cloud.githubusercontent.com/assets/204350/6189676/800ac5c4-b36a-11e4-9ffd-c5f0674493cf.png)

We need to figure out how to either:

* Detect and filter those out when content is saved
* Extend the javascript escaping to remove such characters at that point so that pages don't break

**Comment by Ravenstine** [comment]
I poked at this a little bit:

I didn't reproduce the error in the exact same way, but I did it by inserting that same character into a news story, saving it, and then refreshing the page.  This breaks the preview javascript(probably others too), and the invalid EOF error occurs.

Since I have already worked on a mechanism to clean up form input before it reaches Rails, I'm wondering if maybe filtering of these characters(that JavaScript counts as line endings) should also be rolled into that mechanism.
